amundsen_application/static/js/components/common/Bookmark/BookmarkIcon/index.tsx

```diff
@@ -62,6 +62,7 @@ export class BookmarkIcon extends React.Component<BookmarkIconProps> {
             'icon ' +
             (this.props.isBookmarked ? 'icon-bookmark-filled' : 'icon-bookmark')
           }
+          alt=""
```

Comment:
Yes, it should definitely be a button. We have more instances of this case, but it hasn't been detected by our ESLint rules.

I will create a ticket for this, I think it would make sense to move the star to use SVGs instead at the same time that we change it into a button.

Reply:
This ticket: https://jira.lyft.net/browse/AMD-1513